server/tracedb/tempodb.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func tempoDefaultPorts() []string {
-	return []string{"9095", ""}
+	return []string{"9095", "443", "1"}
```

Comment:
443 and 1 (using https://tempo-us-central1.grafana.net) should be also valid ports

Reply:
I've never seen port 1 being used. Is there any docs about this port?

Reply:
For some reason the url parser uses `1` as port when using something like `https://tempo-us-central1.grafana.net/`

Reply:
![Screen Shot 2024-01-10 at 1 57 31 p m](https://github.com/kubeshop/tracetest/assets/11051031/c554c8dc-d2f8-4076-906a-e419defa72bb)

Reply:
That looks like a bug in the URL parsing library. Do you know why we're using an URL parser other than the go's stdlib one?

Reply:
Yeah the standard makes it worse in some cases, here's a table
![Screen Shot 2024-01-10 at 2 00 28 p m](https://github.com/kubeshop/tracetest/assets/11051031/9052af87-7999-4c66-b873-c98e8bac65e6)

Reply:
I see. That makes sense. What I would do is handle the case of port "1" and replace it by a correct default port, for example:

```
if port == 1 {
  if schema == "http" {
    port = 80
  }
  if schema == "https" {
    port = 43
  }
}
```